src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -535,6 +535,53 @@ public void softDeleteChangedOverlappingRows(Dataset dataset,
         bigQueryProject.query(sql);
     }
 
+    public void softDeleteRowsWithNoOverlap(Dataset dataset,
+                                            Table targetTable,
+                                            String stagingTableName) {
+        /*
+         * INSERT INTO softDeleteTableName
+         * (PDAO_ROW_ID_COLUMN)
+         * SELECT PDAO_ROW_ID_COLUMN
+         * FROM targetTable
+         * EXCEPT DISTINCT
+         * (SELECT PDAO_ROW_ID_COLUMN from stagingTableName)
```

Comment:
This could also `UNION SELECT PDAO_ROW_ID_COLUMN FROM softDeleteTableName` to prevent duplicate entries in the soft-delete list. I'm not sure how to judge the trade-off between scanning the SD table on every overwrite vs. paying for duplicate rows in the table over time.

Reply:
The "right" answer would be to judge based on the rate that the SD table grows, how frequently resets/overwrites happen, how many duplicate rows we expect over time, cost of storage vs compute..etc. but since most of these are dataset-dependent we should just pick a way.

Intuitively, I feel like we should be avoiding duplicate rows at the cost of more SD table scans.